src/App/routes/getting-started.js

```diff
@@ -6,14 +6,14 @@ module.exports = [
             rotate: true
         },
         path: "/get-started",
-        entryCardText: "Learn more about how to setup and use our system",
         redirect: "/get-started/introduction",
         appFolder: "GettingStarted",
         routes: [
             {
                 title: "Introduction",
                 path: "/get-started/introduction",
-                componentPath: "getting-started/Introduction"
+                componentPath: "getting-started/Introduction",
```

Comment:
*nitpick*
Consider renaming folder and component path to "get-started"

Reply:
🚀  Donolini habibi